libcloudforensics/providers/aws/forensics.py

```diff
@@ -136,8 +139,13 @@ def CreateVolumeCopy(zone: str,
       snapshot.aws_account = destination_account
       snapshot = snapshot.Copy(delete=True, deletion_account=source_account)
 
+    kwargs = {'tags': tags}  # type: Dict[str, Any]
+    if tags and tags.get('Name'):
+      kwargs['volume_name'] = tags['Name']
+    else:
+      kwargs['volume_name_prefix'] = 'evidence'
```

Comment:
I don't know how I feel about using `kwargs` for a single parameter here (it was a bit confusing to me because I was looking for other parts of the code that modified it too). It might be more clear to call the function in the if blocks directly.

Reply:
Changed